pkg/version/helpers.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+func CreateAndCheckNotConstrained(baseVersion *semver.Version, version string) (bool, error) {
```

Comment:
What does this function create, besides a temporary variable that I, the caller, do not care in the slightest about? ^^ This could be shortened to just `IsConstrained`, no?

Reply:
Definitely a smarter name, yes.